src/basic/include/net/if.h

```diff
@@ -1,6 +1,7 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 #pragma once
 
+#include <features.h>
```

Comment:
Just add a forward definition if one is needed

Reply:
Forward definition? How?
Note, we already explicitly includes features.h in src/basic/include/sys/mount.h, to make `__THROW` defined.
Each header should be self contained. So, including features.h should be desired.

Reply:
> Note, we already explicitly includes features.h in src/basic/include/sys/mount.h, to make __THROW defined.

So this is not needed and can be dropped, then